package.json

```diff
@@ -57,18 +58,23 @@
     "@babel/runtime-corejs3": "^7.8.7",
     "@blockcerts/explorer-lookup": "^1.2.1",
     "@blockcerts/hashlink-verifier": "^1.3.0",
-    "@blockcerts/schemas": "^3.0.5",
+    "@blockcerts/schemas": "^3.2.1",
+    "@bloomprotocol/ecdsa-secp256k1-signature-2019": "github:lemoustachiste/ecdsa-secp256k1-signature-2019",
+    "@bloomprotocol/ecdsa-secp256k1-verification-key-2019": "github:lemoustachiste/ecdsa-secp256k1-verification-key-2019",
     "@decentralized-identity/did-common-typescript": "^0.1.19",
-    "@digitalbazaar/did-method-key": "github:lemoustachiste/did-method-key#convert-to-module",
     "@digitalbazaar/ed25519-signature-2020": "github:lemoustachiste/ed25519-signature-2020#convert-to-module",
     "@digitalbazaar/ed25519-verification-key-2020": "github:lemoustachiste/ed25519-verification-key-2020#convert-to-module",
+    "@transmute/did-key-ed25519": "^0.3.0-unstable.8",
+    "@transmute/did-key-secp256k1": "^0.3.0-unstable.8",
+    "@transmute/ed25519-key-pair": "^0.7.0-unstable.63",
     "@trust/keyto": "^1.0.1",
     "@vaultie/lds-merkle-proof-2019": "0.0.8",
+    "base58-universal": "github:lemoustachiste/base58-universal#fix/hoisting-error-alphabet",
```

Comment:
linked to this: https://github.com/digitalbazaar/base58-universal/pull/7

With this instruction we overwrite all the deps that rely on this package to use our version which does not have an error in jest.